pandas/core/generic.py

```diff
@@ -2,9 +2,8 @@
 from __future__ import annotations
 
 import collections
-from datetime import timedelta
+import datetime as _dt
```

Comment:
keep this one as `dt` it's

https://github.com/pandas-dev/pandas/blob/1f6e44a2354fa53b724eadb2c52ee84b02300500/pandas/core/generic.py#L9470-L9472

that needs changing to `_dt`